lib/auth/clt.go

```diff
@@ -2670,6 +2650,73 @@ func (c *Client) Ping(ctx context.Context) (proto.PingResponse, error) {
 	return *rsp, nil
 }
 
+// ClientServices implements IClientServices interface.
+type ClientServices struct {
+	roundtrip.Client
+}
+
+// PostJSON is a generic method that issues http POST request to the server
+func (c *ClientServices) PostJSON(
```

Comment:
don't wrap this line https://github.com/golang/go/wiki/CodeReviewComments#line-length
similarly for other methods below

Reply:
revised